helm/sematic-server/values.yaml

```diff
@@ -1,32 +1,33 @@
 auth:
   enabled: true
-  google_oauth_client_id: 977722105393-257kdkrc5dfbpu0jcsd8etn1k4u4q4ut.apps.googleusercontent.com
-  authorized_email_domain: sematic.ai
+  google_oauth_client_id: xxxxxxxx.apps.googleusercontent.com # REPLACE ME
```

Comment:
Just for my education to understand how this works (should not block PR):

- How are users supposed to override these values? Are they supposed to modify this file? or provide them on the CLI? (`--values`)

- In case a user accidentally kicked off the helm install with the default values. Is it supposed to work? If not, do we have a plan to show errors and mitigate instructions?

Reply:
* This is built into Helm itself.  Users can override these values in several different ways.  CLI flags can be used for one-off overrides.  They can also pass in their own values file via the `--values` flag and override several options at once.
* It will not work.  There is more setup that they need to do before they install, such as specifying their own values, setting up the namespace, creating secrets, creating the Google OAuth app, etc.  This will/should be documented better once we have confirmed that this actually works for Voxel first, since they are the only currently interested user of this, and we're using them to trial it.

Reply:
Thanks for the explanation. 

Cannot agree more on the `This will/should be documented better` part. Glad to know that we have follow-up items planned. 👍